### PR TITLE
executor: Remove Next() in IndexLookUpExecutor

### PR DESCRIPTION
**Pull request by lhyPingcap**
modify distsql.go

### CODE REVIEW

**Comment by winkyao** [comment]
The description changed to `Remove Next() int IndexLookUpExecutor` maybe better?

File: executor/distsql.go [hunk 1]
```diff
@@ -670,26 +670,6 @@ func (e *IndexLookUpExecutor) Close() error {
 	return nil
 }
 
-// Next implements Exec Next interface.
-func (e *IndexLookUpExecutor) Next(ctx context.Context) (Row, error) {
```
Comment (winkyao) [inline executor/distsql.go]: lookupTableTask.getRow is useless, please remove it.

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**